tests/scripts/migrate_duplicated_community_channels_and_groups.spec.ts

```
@@ -0,0 +1,488 @@
+
+describe('Migration Script: migrate_duplicated_community_channels_and_groups', () => {
+  let sandbox: any;
+  let exitStub: any;
+  let communityFindStub: any;
+  let userFindByIdStub: any;
+  let isGroupAdminStub: any;
+  let connectStub: any;
+  let createInterfaceStub: any;
+  let mockRl: any;
+  let botStub: any;
+  let consoleLogStub: any;
```

ESLint

[error] 'consoleLogStub' is assigned a value but never used. Allowed unused vars must match /^_/u.

(`@typescript-eslint/no-unused-vars`)

---

[error] 'consoleLogStub' is assigned a value but never used.

(no-unused-vars)